types/node-easy-cert/node-easy-cert-tests.ts

```diff
@@ -0,0 +1,82 @@
+import * as CertManager from 'node-easy-cert';
```

Comment:
`CertManager` is being imported as namespace which is not callable. Instead use `import CertManager = require('node-easy-cert');`

Reply:
I'm not sure that it was needed since `CertManager` is a mix of a function, a class and a namespace.
Maybe in old TypeScript versions...

To avoid confusion, I fixed this in last commit.